app/src/main/java/org/fossasia/activities/MainActivity.java

```diff
@@ -268,6 +268,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    private void startDownloadSchedule() {
```

Comment:
Yes @batraabhishek I know that but I made this function with the intention of showing the splash screen as I thought it would look better and would show the user that something is taking place instead of just the toast notification. It looks good this way.